OpenData/secular printed material 1750-1900/Mannanan & shen venn Vanninagh/manifest.json.txt

```diff
@@ -0,0 +1,9 @@
+{
+	"created": "1874-01-01",
+	"ident": "Mannanan-as-shenn-ven-Vanninagh",
+	"name": "Mannanan & shenn ven Vanninagh",
+	"notes": "By Thos. Kelly, Bridge St. Peel., translated into Manx By Rev. J.T.Clarke. Transcribed & translated; R.Teare 2021, from Mona's Herald, Thursday, January 01, 1874; Page: 5", 
+	"translated": "Rob Teare 2021",
+	"source":  "Manks Advertiser, Thursday, March 09, 1826; Page: 4",
```

Comment:
@robteare I've changed the source to `Mona's Herald`. I don't see this in the Advertiser.